README.md

```diff
@@ -1,105 +1,186 @@
+[FINOS IP Policy]: https://github.com/finos/community/blob/master/governance/IP-Policy.pdf
+[FINOS standards governance]: https://github.com/finos/community/tree/master/governance/Standards-Projects
+
+## The Product Management Committee
+
+The [FDC3 Product Management Committee] (PMC) is responsible for co-ordinating the work of the FDC3 Standard Working Group, as well as FDC3-related planning and communications. For more context see the [FDC3 Archive].
+
+
+[FDC3 Product Management Committee]: https://github.com/finos/FDC3/issues?q=label%3A%22PMC+Meeting%22
+[FDC3 Archive]: https://finosfoundation.atlassian.net/wiki/spaces/FDC3/overview
+
+### Meetings
+The PMC meets every second Friday at 9:30am ET / 2:30pm GMT. Find the next meeting, including meeting details, on the [FINOS Project Meetings Calendar].
+
+### Mailing list
+You can get in touch with the FDC3 PMC by sending an email to [fdc3-pmc@finos.org]. Join the PMC Mailing list by sending an email to [fdc3-pmc+subscribe@finos.org].
+
+[fdc3-pmc@finos.org]: mailto:fdc3-pmc@finos.org
+[fdc3-pmc+subscribe@finos.org]: mailto:fdc3-pmc+subscribe@finos.org
+
+# Contributing
+
+If you'd like to contribute code to this repository:
+
+1. Fork it (<https://github.com/finos/FDC3/fork>)
+2. Create your feature branch (`git checkout -b feature/fooBar`)
+3. Read our [contribution guidelines](CONTRIBUTING.md) and [Community Code of Conduct](https://www.finos.org/code-of-conduct)
+4. Commit your changes (`git commit -am 'add some fooBar'`)
+5. Push to the branch (`git push origin feature/fooBar`)
+6. Create a new Pull Request
 
-#### Use Cases
-* Continue to process new and existing FDC3 Use Cases
-* Assess extending the use cases working group to additional FINOS programs
+_NOTE:_ Commits and pull requests to FINOS repositories will only be accepted from those contributors with an active, executed Individual Contributor License Agreement (ICLA) with FINOS OR who are covered under an existing and active Corporate Contribution License Agreement (CCLA) executed with FINOS. Commits from individuals not covered under an ICLA or CCLA will be flagged and blocked by the FINOS Clabot tool. Please note that some CCLAs require individuals/employees to be explicitly named on the CCLA.
 
-### Backlog Beyond Q2
-* Define the next set of FDC3 Context Data types in conjunction with/based on Financial Objects working group work.
-* Work with Financial Objects and other FINOS programs to reach consensus on how to make standardized taxonomies available for consumption, both for viewing/editing and programmatically e.g. for validation. 
-* Create comprehensive examples of use cases that show how to use the various FDC3 standards together to enable interoperability.
-* App Directory Identity and authentication
+*Need an ICLA? Unsure if you are covered under an existing CCLA? Email [help@finos.org](mailto:help@finos.org).*
 
-## License
+# License
 
 Copyright 2017 FDC3
```

Comment:
> Copyright 2017-2021 FINOS

?

Reply:
Yeah, not sure, need FINOS guidance on this one.

Reply:
@mattjamieson that is OK. I'll send a PR a later on if I have a different proposal.